src/userscores/qml/MuseScore/UserScores/internal/TimeSignatureFraction.qml

```diff
@@ -18,6 +18,7 @@ Row {
         id: control
 
         implicitWidth: 68
+        anchors.verticalCenter: parent.verticalCenter
```

Comment:
note for future - prefer to put anchors right after id, it's kind of widespread convention in QML